src/app/screens/ConfirmPayment/index.tsx

```diff
@@ -84,6 +85,7 @@ function ConfirmPayment(props: Props) {
 
   return (
     <div>
+      <ToastContainer />
```

Comment:
is this needed here? this should already be included from the prompt/popup screen?

Reply:
We don't need this extra here.

Reply:
Without it imported here it does not show up on the popup screen.

Reply:
I think this needs to go into the `Prompt/Prompt.tsx` ? 
similar to the Popup/Popup.tsx ?

Reply:
pushed some changes.

Reply:
> I think this needs to go into the `Prompt/Prompt.tsx` ? similar to the Popup/Popup.tsx ?

Thanks, I missed these comments over the weekend